ui/components/MesheryPerformanceComponent.js

```diff
@@ -117,8 +117,15 @@ class MesheryPerformanceComponent extends React.Component {
 
   handleChange = (name) => (event) => {
     if (name === 'url' && event.target.value !== '') {
+      const compulsoryProtocolValidUrlPattern = new RegExp('(^(http|https|nats|tcp)?:\\/\\/)' // compulsory protocol
```

Comment:
thanks, @shubhamgupta2956. Have a quick check on the use of `?`. It might produce a positive match in the event the user doesn't enter a protocol, like this: `://example.com/endpoint`

Reply:
Yes, it is allowing this. I have updated the commit and removed the `?`.